pulp_container/tests/functional/api/test_rbac_repo_versions.py

```diff
@@ -217,25 +214,13 @@ def test_cross_repository_blob_mount(
     # Cleanup
     add_to_cleanup(container_namespace_api, distribution.namespace)
 
-    # Test that admin can cross mount
-    for i, blob in enumerate(blobs, start=1):
+    # Test that an admin can cross mount but do not perform any further assertions because
+    # the blobs are committed only after uploading the final tagged manifest
```

Comment:
I think you should be able to do a get on the mounted blob, as it's now considered to be part of the target repo.

Reply:
And, I have found out I need to update the registry.py code because we query `ContentArtifact` against "repository_version.get_content" only (See the `get_by_digest` method).

Reply:
I don't think this is correct. You are not supposed to stream the content to the client if it is not fully committed yet to the repo which happens in the finalize_new_version.
It is ok and correct to give a 200 on handle_safe_methods though.